certified-connectors/InEight/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3049 @@
+                      }
+                    }
+                  },
+                  "AdditionalPayrollInstructionCodeOverridden": {
+                    "type": "boolean",
+                    "description": "If the Allow Additional Payroll Instruction Code for the employee has been modified (changed or removed) for the time card for any reason, this field will show as true.",
+                    "example": "false"
+                  },
+                  "IsActive": {
+                    "type": "boolean",
+                    "description": "If the timecard is active, this field will show true",
+                    "example": "true"
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/api/TimeCardConfirmation": {
+      "post": {
+        "description": "The Time Card Confirmation integration allows external systems to report on the success or failure to process time cards obtained from Time Center and update a status to support a business workflow.",
+        "operationId": "TimeCardConfirmation_Import",
+        "summary": "TimeCardConfirmation_Import",
+        "parameters": [
+          {
+            "name": "X-IN8-TENANT-PREFIX",
+            "x-ms-summary": "This field contains Tenant Prefix",
+            "description": "Tenant Prefix",
+            "in": "header",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "Ocp-Apim-Subscription-Key",
+            "x-ms-summary": "This field contains Tenant Subscription key",
+            "description": "Subscription Key",
+            "in": "header",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "TimeCardId": {
+                    "type": "string",
+                    "description": "TimeCardId"
+                  },
+                  "Version": {
+                    "type": "string",
+                    "description": "Version"
+                  },
+                  "Status": {
+                    "type": "string",
+                    "description": "Status"
+                  },
+                  "Errors": {
+                    "type": "array",
+                    "items": {
+                      "type": "string"
+                    },
+                    "description": "Errors"
+                  }
+                }
+              }
+            }
+          }
+        ],
+        "consumes": [
+          "application/json",
+          "text/plain"
+        ],
+        "produces": [
+          "application/json",
+          "text/plain"
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {}
```

Comment:
@InEightDevOps Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.